ccd-definition/CaseEvent.json

```diff
@@ -59,6 +59,20 @@
     "ShowEventNotes": "N",
     "EndButtonLabel": "Save and continue"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "amendChildren",
+    "Name": "Children",
+    "Description": "Amending the children for the case",
+    "DisplayOrder": 4,
+    "PreConditionState(s)": "Submitted",
+    "PostConditionState": "Submitted",
+    "SecurityClassification": "Public",
+    "ShowSummary": "N",
+    "ShowEventNotes": "Y",
+    "EndButtonLabel": "Save and continue"
+  },
```

Comment:
let's group these events with the other submitted events? We should also take a look at display order, was there any input from design about where these events should sit in the drop down? Will be easier to see the numbers when they are bunched

Reply:
Thanks I will check with them.